prestodb/hdp2.6-hive-kerberized/Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM prestodb/hdp2.6-hive:unlabelled
 MAINTAINER Presto community <https://prestodb.io/community.html>
 
 # INSTALL KERBEROS
-RUN yum install -y krb5-libs krb5-server krb5-worqkstation \
+RUN yum install -y krb5-libs krb5-server krb5-workstation \
```

Comment:
Does `yum install` ignore uninstallable packages and return exit code 0?

See
* comments under https://stackoverflow.com/a/8447232/65458
* https://access.redhat.com/solutions/321603 (rhel 5 example)